partcad_skyline/skyline.py

```
@@ -0,0 +1,146 @@
+class Config(dict):
+    api: str
+
+
+def serialize_date_time(obj: Union[datetime.date, datetime.datetime]) -> str:
+    if isinstance(obj, (datetime.date, datetime.datetime)):
+        return obj.isoformat()
+    raise TypeError(f"Type {type(obj)} not serializable")
+
+
+class GitRepoManager:
+    def __init__(self, repo_url, clone_dir):
```

Ruff (0.8.0)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)